### DISCUSSION TITLE
calling `yabai` window manager from `cmd-button` doesn't work

### DISCUSSION THREAD

**Discussion by molleweide** [kmonad/kmonad#422] [Q&A] [answered]
I am going to try to create a dedicated layer for hosting my window manager commands.
I am using yabai on macos. I will report back if it works or doesn't :P

**Comment by molleweide** [comment]
okay so now I have tried some yabai commands and this is what happens when I call yabai commands from kmonad:
```shell
╰ kmor
Password:
connected
yabai-msg: failed to connect to socket..
yabai-msg: failed to connect to socket..
yabai-msg: failed to connect to socket..
yabai-msg: failed to connect to socket..
yabai-msg: failed to connect to socket..
yabai-msg: failed to connect to socket..
```
every time I call a yabai command it returns socket fail.

This is an example of my mappings:

```clojure
  T34 (cmd-button "yabai -m space --layout stack")
```

Does anyone know what this means??
Is this a question that is appropriate here (I think so right?) or should I go and ask under yabai here on github??

**Comment by slotThe** [comment, marked as answer]
For completeness sake: this is https://github.com/kmonad/kmonad/issues/423 and https://github.com/kmonad/kmonad/issues/426
